### PR TITLE
Update desyn mode chian

### PR DESCRIPTION
**Pull request by santinoYoung**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm exports TVL: 
        
 
 
    merlin                    83.84 M
    
    total                    83.84 M

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm1 exports TVL: 
        
 
 
    btr                       53.21 M
    mode                      0
    
    total                    53.21 M

**Comment by llamatester** [comment]
The adapter at projects/desyn exports TVL: 
        
 
 
    ethereum                  5.88 M
    arbitrum                  345
    
    total                    5.88 M

File: projects/desyn-farm1/index.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 const { getLogs } = require('../helper/cache/getLogs')
 
 module.exports = {
-  methodology: 'On-chain restaking',
+  methodology: 'Product: On-chain restaking<br/>TVL: It includes above<br/>Revenue: Staking rewards, management fees, performance fees',
```
Comment (g1nt0ki) [inline projects/desyn-farm1/index.js:4]: I am tired of reverting this every time

Comment (santinoYoung) [reply projects/desyn-farm1/index.js:4]: hi，sorry，guys，I'll pay attention next time.

**Comment by llamatester** [comment]
The adapter at projects/desyn-farm1 exports TVL: 
        
 
 
    btr                       55.27 M
    mode                      0
    
    total                    55.27 M

**Pull request merged**